### ISSUE TITLE
Samples/Heatmap Layers/Heatmap_Features.html doesn't display a heatmap

### ISSUE THREAD

**Issue by jkpratt** [Microsoft/BingMapsV8CodeSamples#28]
I tried running the heatmap at https://bingmapsv8samples.azurewebsites.net/#Heatmap_Features, when you press update  heatmap , nothing happens. I tried setting the options in many ways but nothing changes. It seems that the heatmap layer isn't dynamically updating neither the locations array nor the heatmapOptions object. Is is possible to dynamically update?

**Comment by jkpratt** [comment]
There is an error with the selector function.  I fixed that error and instead of using 0 location points upon start I used 5000 and the heatmap displayed. But changing the options has no affect what so ever. Can a heatmap be dynamically updated.

VM425:117 Uncaught TypeError: Failed to execute 'item' on 'NodeList': 1 argument required, but only 0 present.
    at getSelectedValue (<anonymous>:117:52)
    at UpdateHeatMap (<anonymous>:63:20)
    at HTMLInputElement.onclick ((index):1)
